lib/active_merchant/billing/gateways/payeezy.rb

```diff
@@ -25,13 +25,21 @@ class PayeezyGateway < Gateway
         'diners_club' => 'Diners Club'
       }
 
+      PARAMS_FOR_TOKENIZATION = {
+        api_key: 'y6pWAJNyJyjGv66IsVuWnklkKUPFbb0a',
+        js_security_key: 'js-6125e57ce5c46e10087a545b9e9d7354c23e1a1670d9e9c7',
+        ta_token: 'NOIW',
+        callback: 'Payeezy.callback',
+        type: 'FDToken'
+      }
```

Comment:
Where does the `api_key` and `js_security_key` come from? Are these sensitive values that should not be hardcoded in?

Reply:
Good catch - at first I thought these were static, public values from https://developer.payeezy.com/payeezy-api/apis/post/transactions-14 but those must be example values - api_key is a credential, and it looks like the gateway needs to activate the ability to store on a per-merchant account basis. That might be how the `js_security_key` is provided?

Reply:
Looking at the Payeezy docs, looks like the `api_key` is the same as the one we use in the `initialize` so we can reference it in the `@options` hash. Also, wondering if we should move the `js_security_key` to the initialize option since it's a credential. Not a required one, just set it's value there if it was passed in.

Reply:
👍 Their documentation is certainly lacking in this area.

@deedeelavinder - It might be worth digging through their admin / management UI to verify how sensitive these two values are.

Reply:
As @davidsantoso pointed out, we can use the `:apikey` used to initialize the gateway. The `js_security_key` is merchant-specific but is not treated as ultra-sensitive in their documentation and support forums. This now allows that parameter to be passed in with options. That will allow backward compatibility for current users who wish to use `store` in the future.